app/src/main/java/com/example/concordia_campus_guide/Models/Rooms.java

```diff
@@ -17,6 +21,47 @@ public Rooms(List<RoomModel> rooms) {
         return Rooms;
     }
 
+    public List<RoomModel> getRoomsByFloor(String floor){
+        ArrayList<RoomModel> list = new ArrayList<>();
+        List<RoomModel> rooms = getRooms();
+        for(RoomModel room: rooms){
+            if(room.getFloorCode().equals(floor)){
+                list.add(room);
+            }
+        }
+        return list;
+    }
```

Comment:
Wouldn't make more sense to have this method in the RoomModel class instead  ? because this class might be deleted later on in the future.

Reply:
Hi, I put it in the Rooms class because when we initialize the app we import all the room data using a JSON and then store it in the rooms attribute of the Rooms object. Thus, this will allow us to bypass querying the database and simply loop through the list as needed.